.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ permissions:
 
 jobs:
   CI:
-    uses: grafana/plugin-ci-workflows/.github/workflows/ci.yml@main
+    uses: grafana/plugin-ci-workflows/.github/workflows/ci.yml@9bacf72016cc3cd1b0fdd0eacf10baccdaf4f87c
```

Comment:
We can pin the version of this workflow, but I don't think we need to for zizmor reasons. When we run https://github.com/grafana/github-action-patcher on this repo, it doesn't complain about this workflow being `@main`.

Reply:
roger that ill revert to main